docroot/themes/custom/epa_intranet/preprocess/paragraph/paragraph__uswds_hero.preprocess.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Implements hook_preprocess_paragraph__uswds_hero().
+ */
+function epa_intranet_preprocess_paragraph__uswds_hero(&$variables) {
+  $variables['attributes']['class'][] = 'usa-hero';
+
+  // We also need to set the inline CSS for the image.
+  if (!$variables['paragraph']->field_uswds_image->isEmpty()) {
```

Comment:
Can you add another "isset" here for $variables['paragraph'] and $variables['paragraph']->field_uswds_image? We get a lot of logs for invalid indexes, in case this hook is ran more than we expect.

Reply:
Updated